plutus-core/src/Language/PlutusCore/Evaluation/Machine/Cek.hs

```diff
@@ -554,9 +555,38 @@ unsafeEvaluateCek runtime = either throw id . extractEvaluationResult . evaluate
 readKnownCek
     :: ( GShow uni, GEq uni, Closed uni, uni `Everywhere` ExMemoryUsage
        , KnownType (Plain Term uni fun) a
-       , Hashable fun, Ix fun, ExMemoryUsage fun
+       , Hashable fun, Ix fun
        )
     => BuiltinsRuntime fun (CekValue uni fun)
     -> Plain Term uni fun
     -> Either (CekEvaluationException uni fun) a
 readKnownCek runtime = evaluateCek runtime >=> readKnown
+
+-- Don't trust this one, use the function from the untyped Cek.
```

Comment:
Why not?

Reply:
Because I didn't intend to calibrate it. Should I?

Reply:
I meant more: explain in the comment. As a reader, it's quite mystifying.